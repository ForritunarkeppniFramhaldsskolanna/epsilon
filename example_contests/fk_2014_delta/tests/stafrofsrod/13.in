15
hckqkbnc
hvernadpuhwck
ngekq
ackslxerrjvclkc
prdgzhpnztvoxykz
vjptxqpfztk
ct
tqznqdies
thcwzqjmw
gyskeiggex
vaj
qc
hlojdandxhbc
spmwavvewksxllfobti
qmvoufr
