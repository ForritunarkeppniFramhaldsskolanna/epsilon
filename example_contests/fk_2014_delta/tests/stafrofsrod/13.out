ackslxerrjvclkc
ct
gyskeiggex
hckqkbnc
hlojdandxhbc
hvernadpuhwck
ngekq
prdgzhpnztvoxykz
qc
qmvoufr
spmwavvewksxllfobti
thcwzqjmw
tqznqdies
vaj
vjptxqpfztk
